### PR TITLE
Added title attribute on role ui element

### PR DESCRIPTION
**Pull request by mkumbobeaty**

### CODE REVIEW

File: src/Stakeholders/components/Roles/List/index.js [hunk 1]
```diff
@@ -168,7 +176,18 @@ class RoleList extends Component {
           onPaginate={nextPage => {
             paginateRoles(nextPage);
           }}
-          onRefresh={refreshRoles}
+          onRefresh={() =>
+            refreshRoles(
+              () => {
+                notifySuccess('Role refreshed successfully');
```
Comment (BenMaruchu) [inline src/Stakeholders/components/Roles/List/index.js]: Role should be Roles

File: src/Stakeholders/components/Roles/List/index.js [hunk 2]
```diff
@@ -168,7 +176,18 @@ class RoleList extends Component {
...
+              },
+              () => {
+                notifyError(
+                  'An Error occurred while refreshing role please contact system administrator'
```
Comment (BenMaruchu) [inline src/Stakeholders/components/Roles/List/index.js]: same here should be roles

File: src/Stakeholders/components/Roles/ListItem/index.js [hunk 3]
```diff
@@ -135,21 +135,27 @@ class RoleListItem extends Component {
...
+          <Col span={10} title="An additional information attached to a role">
+            {description}
+          </Col>
           <Col span={3}>
             {isHovered && (
               <Fragment>
                 <Icon
                   type="edit"
-                  title="Update Role"
+                  title="Click to update Role"
```
Comment (BenMaruchu) [inline src/Stakeholders/components/Roles/ListItem/index.js]: was this supposed to be changed?

**Review by BenMaruchu — CHANGES_REQUESTED** [review]

File: src/Stakeholders/components/Roles/ListItem/index.js (hunk 3 above)
Comment (mkumbobeaty) [reply src/Stakeholders/components/Roles/ListItem/index.js]: yes lally told me to put that way

Comment (BenMaruchu) [reply src/Stakeholders/components/Roles/ListItem/index.js]: ok I talked to Lally so can we return these back to original titles without click word? @BenMaruchu

**Review by BenMaruchu — APPROVED** [review]

**Pull request merged**